rollup/app/lwc/rollupForceRecalculation/rollupForceRecalculation.html

```diff
@@ -1,77 +1,93 @@
+      SObject type:
+    </div>
+    <lightning-input type="toggle" label="Run off of CMDT?" onchange={handleToggle}></lightning-input>
     <lightning-layout vertical-align="center">
       <div class="slds-grid slds-grid_vertical slds-gutters_small slds-grid_align-center" role="list">
-        <lightning-input
-          class="slds-col slds-form-element slds-form-element_horizontal"
-          type="text"
-          label="Calc Item SObject API Name"
-          name="calcItemSObjectName"
-          onchange={handleChange}
-          required
-        >
-        </lightning-input>
-        <lightning-input
-          class="slds-col slds-form-element slds-form-element_horizontal"
-          type="text"
-          label="API Name of the rollup field on calc item"
-          name="opFieldOnCalcItem"
-          onchange={handleChange}
-          required
-        >
-        </lightning-input>
-        <lightning-input
-          class="slds-col slds-form-element slds-form-element_horizontal"
-          type="text"
-          label="API Name of the lookup field on calc item"
-          name="lookupFieldOnCalcItem"
-          onchange={handleChange}
-          required
-        >
-        </lightning-input>
-        <lightning-input
-          class="slds-col slds-form-element slds-form-element_horizontal"
-          type="text"
-          label="API Name of the lookup field on the lookup object"
-          name="lookupFieldOnLookupObject"
-          onchange={handleChange}
-          required
-        >
-        </lightning-input>
-        <lightning-input
-          class="slds-col slds-form-element slds-form-element_horizontal"
-          type="text"
-          label="API Name of the Rollup field on the lookup object"
-          name="rollupFieldOnLookupObject"
-          onchange={handleChange}
-          required
-        >
-        </lightning-input>
-        <lightning-input
-          class="slds-col slds-form-element slds-form-element_horizontal"
-          type="text"
-          label="Lookup SObject API Name"
-          name="lookupSObjectName"
-          onchange={handleChange}
-          required
-        >
-        </lightning-input>
-        <lightning-input
-          class="slds-col slds-form-element slds-form-element_horizontal"
-          type="text"
-          label="Rollup Operation Name (SUM/MIN/MAX/COUNT/COUNT_DISTINCT/CONCAT/CONCAT_DISTINCT/AVERAGE)"
-          name="operationName"
-          onchange={handleChange}
-          required
-        >
-        </lightning-input>
-        <lightning-textarea
-          class="slds-col slds-form-element slds-form-element_horizontal"
-          onchange={handleChange}
-          label="Where Clause SOQL Filters (Optional)"
-          name="potentialWhereClause"
-        ></lightning-textarea>
+        <template if:true={isCMDTRecalc}>
+          <lightning-combobox
```

Comment:
Scope-creep idea - would it be possible to show a table of the matching CMDT records (based on the selected `Calc Item`)? In my org, I couldn't remember which CMDT records I actually had setup on `Contact`, so I still had to go under `Setup --> Custom Metadata Types` (screenshot below). Displaying a similar table within the `lwc` would save a little extra clicking around.

![image](https://user-images.githubusercontent.com/1267157/112763285-af231b00-8fb8-11eb-9aec-ac3a1be6e0ec.png)

Reply:
I kicked this over to #62 - I think this is a great enhancement, but won't make it into this release.